packages/react/src/__tests__/__snapshots__/storyshots.test.js.snap

```diff
@@ -3506,6 +3506,18 @@ exports[`Storyshots Components|Dotcom Shell Default 1`] = `
                                             classname="bx--card__img"
                                             defaultSrc="https://dummyimage.com/600x600/ee5396/161616&text=1:1"
                                           >
+<<<<<<< HEAD
```

Comment:
Probably the editor showed false nagative on remaining merge conflicts...?

Reply:
Hm, I think so? I'm not seeing that on my end.

Reply:
Nope, you were right @asudoh 😄